jsaddle/src/Language/Javascript/JSaddle/Object.hs

```diff
@@ -444,7 +444,7 @@ foreign import javascript unsafe "$r = function () { $1(this, arguments); }"
     makeFunctionWithCallback :: Callback (JSVal -> JSVal -> IO ()) -> IO Object
 #endif
 
--- ^ Make a JavaScript function object that wraps a Haskell function.
+-- | Make a JavaScript function object that wraps a Haskell function.
```

Comment:
This doc comment was dropped from the haddocks because it refers to the previous item (-- ^) instead of the following (-- |).